qui/updater/updater.py

```diff
@@ -418,6 +419,12 @@ def parse_args(args, app):
                              'Interaction will be required in the event '
                              'of an update error.')
 
+    parser.add_argument('--show-advanced-settings', action='store_true',
```

Comment:
See comment here: https://github.com/QubesOS/qubes-issues/issues/9029#issuecomment-2435510421

Reply:
This is done. The updater will show the advanced settings by default and no CLI argument is needed.